### PR TITLE
Pr/6

### PR DESCRIPTION
**Pull request by bjswoboda**

### CODE REVIEW

File: ElasticBox/partner-cloud-integration-azure-new.md [hunk 1]
```diff
@@ -1,36 +1,72 @@
...
+### Steps
+
+Your goal is to set up a new ElasticBox provider that has been designed to create an Azure Customer account within the scope of CenturyLink's responsibility.
+
+1. Log into ElasticBox.
+2. Select the Providers tab
+3. Select New
+4. Select "Azure Resource Manager"
+
+![Azure Resource Manager Provider](../images/CINT_New_ARM1.png)
```
Comment (srveit) [inline ElasticBox/partner-cloud-integration-azure-new.md]: Change link to `../images/ElasticBox/CINT_New_ARM1.png`

File: ElasticBox/partner-cloud-integration-azure-new.md [hunk 2]
```diff
@@ -1,36 +1,72 @@
...
+
+5. Select the "Create a new Azure customer account" option
 
+![Create New Azure Account](../images/CINT_New_ARM2.png)
```
Comment (srveit) [inline ElasticBox/partner-cloud-integration-azure-new.md]: Change link to `../images/ElasticBox/CINT_New_ARM2.png`

File: ElasticBox/partner-cloud-integration-azure-new.md [hunk 3]
```diff
@@ -1,36 +1,72 @@
...
 
-* Email support@elasticbox.com
+6. Provide a name for the provider that identifies it for your purposes.
+7. Click the "Connect a CLC Account" button and provide your CenturyLink Cloud Account Administrator credentials
 
+![Create New Azure Account](../images/CINT_New_ARM3.png)
```
Comment (srveit) [inline ElasticBox/partner-cloud-integration-azure-new.md]: Change link to `../images/ElasticBox/CINT_New_ARM3.png`

File: ElasticBox/partner-cloud-integration-azure-new.md (hunk 1 above)
Comment (bjswoboda) [inline ElasticBox/partner-cloud-integration-azure-new.md]: done

**Pull request merged**